src/os/core/linux/os_core_linux.c

```diff
@@ -4,6 +4,7 @@
 ////////////////////////////////
 //~ rjf: Helpers
 
+#include <asm-generic/errno.h>

```

Comment:
If errno.h header is needed then it should be `#include <errno.h>` not `<asm-generic/errno.h>`